src/views/Dashboard/Customer/Home/MyTest/TestResult.vue

```diff
@@ -0,0 +1,181 @@
+<template lang="pug">
+  div
+    v-container
+      ui-debio-card(width="100%")
+        v-row.resultBody
+          v-col(cols="12" md="9")
+            ui-debio-card(width="100%" height="100%" class="mt-2")
+              template
+                v-progress-linear(
+                  v-if="resultLoading"
+                  indeterminate
+                  color="primary"
+                )
+                v-card-text
+                  embed(
+                    :src="reportResult"
+                    type="application/pdf"
+                    v-if="isDataPdf"
+                    scrolling="auto"
+                    height="500px"
+                    width="100%"
+                  )
+                  //- span(v-else) {{reportResult}}
+                  div
+                    span {{dummyResult.title}} 
+                    br
+                  div
+                    span {{dummyResult.subTitle}}
+                  div
+                    span {{dummyResult.content}}
+          v-col(cols="12" md="3")
+            //- div(class="mb-2")
+            div.buttonSection(v-for="(file, index) in files" :key="file.name")
+              ui-debio-card(
+                :title="file.fileTitle"
+                :sub-title="file.fileSubTitle"
+                tiny-card
+                with-icon
+                @click="actionDownload(file.fileType)"
+              )
+                ui-debio-icon(
+                  slot="icon"
+                  size="33"
+                  :icon="downloadIcon"
+                  stroke
+                  color="#c400a5"
+                )
+            //- ui-debio-card(
```

Comment:
Please remove comments like these.

Reply:
i have deleted this comment @agustinustheo